--- Makefile
+++ Makefile
@@ -29,6 +29,6 @@ post-patch:
 
 post-install:
 	${STRIP_CMD} ${STAGEDIR}${PYTHONPREFIX_SITELIBDIR}/grpc/_cython/cygrpc*.so
-	${REINPLACE_CMD} -e 's|^.*so$$|${PYTHONPREFIX_SITELIBDIR}/grpc/_cython/cygrpc${PYTHON_TAG}.so|' ${_PYTHONPKGLIST}
+	${REINPLACE_CMD} -e 's|^.*so$$|${PYTHONPREFIX_SITELIBDIR}/grpc/_cython/cygrpc${PYTHON_SOABI}.so|' ${_PYTHONPKGLIST}
 
 .include <bsd.port.mk>
