--- Makefile
+++ Makefile
@@ -1,5 +1,5 @@
 PORTNAME=	libvirt
-PORTVERSION=	12.5.0
+PORTVERSION=	12.6.0
 CATEGORIES=	devel
 MASTER_SITES=	http://libvirt.org/sources/python/ \
 		ftp://libvirt.org/libvirt/python/
--- distinfo
+++ distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1782910138
-SHA256 (libvirt_python-12.5.0.tar.gz) = 25ea2261e0d6cab9d004b700911d88b57911b14528ed04b3487a617ec2b4cda9
-SIZE (libvirt_python-12.5.0.tar.gz) = 245493
+TIMESTAMP = 1785755805
+SHA256 (libvirt_python-12.6.0.tar.gz) = f390abff81d4afd6c33157ade82912863b87de9c35172e9b0c9f0dfcdc09b139
+SIZE (libvirt_python-12.6.0.tar.gz) = 252693
